### ISSUE TITLE
[JENKINS-11769] Add ability to require multiple approvals before promotion

### ISSUE THREAD

**Issue by jenkins-infra-bot** [jenkinsci/promoted-builds-plugin#427]
<p>It would be nice to have jenkins promote a build only after multiple stake holders have approved it.</p>

<p>In our case we want Dev, QA, and PM to sign off on build before it's made available to a customer.</p>

<p>There is another use case described in this thread: <a href="http://groups.google.com/group/jenkinsci-users/browse_thread/thread/a098ad92c3755806" class="external-link" target="_blank" rel="nofollow noopener">http://groups.google.com/group/jenkinsci-users/browse_thread/thread/a098ad92c3755806</a></p>

---
<details><summary><i>Originally reported by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=jminne">jminne</a>, imported from: <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-11769" target="_blank">Add ability to require multiple approvals before promotion</a></i></summary>
<i><ul>
<li><b>status</b>: Open
<li><b>priority</b>: Major
<li><b>component(s)</b>: promoted-builds-plugin
<li><b>resolution</b>: Unresolved
<li><b>votes</b>: 11
<li><b>watchers</b>: 5
<li><b>imported</b>: 2025-12-07
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>It would be nice to have jenkins promote a build only after multiple stake holders have approved it.</p>

<p>In our case we want Dev, QA, and PM to sign off on build before it's made available to a customer.</p>

<p>There is another use case described in this thread: <a href="http://groups.google.com/group/jenkinsci-users/browse_thread/thread/a098ad92c3755806" class="external-link" target="_blank" rel="nofollow noopener">http://groups.google.com/group/jenkinsci-users/browse_thread/thread/a098ad92c3755806</a></p></pre>
</details>
</details>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>thamps:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-11769?focusedId=179825&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-179825">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Why is this feature not possible?</p>
</pre>
</details>

</ul>
</details>

<p>Why is this feature not possible?</p>
